internal/tfimport/importer/resources.go

```diff
@@ -16,11 +16,13 @@ package importer
 
 import (
 	"bufio"
+	"bytes"
 	"fmt"
 	"io"
 	"log"
 	"os"
 	"strings"
+	"text/template"
 )
 
 // ProviderConfigMap is a type alias to make variables more readable.
```

Comment:
Consider adding more documentation around this type. It's getting a bit overloaded as now loadFields can take rc.Change.After, pcv, i.Defaults as ProviderConfigMap.